parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileWriter.java

```diff
@@ -950,7 +950,24 @@ public long getPos() throws IOException {
   }
 
   public long getNextRowGroupSize() throws IOException {
-    return alignment.nextRowGroupSize(out);
+    return getNextRowGroupSize(out.getPos());
+  }
+
+  public long getNextRowGroupSize(long outPos) {
```

Comment:
Not sure if these methods need to be public. If yes, please comment at least that these are for internal use only.

Reply:
Good point, made them package-private.